config/manager/manager.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
     app.kubernetes.io/component: manager
     app.kubernetes.io/created-by: lightspeed-operator
     app.kubernetes.io/part-of: lightspeed-operator
+    openshift.io/cluster-monitoring: "true"
```

Comment:
@raptorsun Should we hard code this - this option comes as  OLM install option. Check this out :https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html/logging/cluster-logging-deploying

Reply:
We can keep it scraped by in-cluster monitoring stack. 
If new requirements arrives, let's see how to make it an option later.

Reply:
The current version of cluster-logging-operator have the cluster monitoring turned on hardcoded, too :) 
![image](https://github.com/openshift/lightspeed-operator/assets/6040960/0a79b4e2-f752-49f0-b94a-b9970c4402f6)